Makefile

```diff
@@ -147,7 +147,7 @@ test: $(GOBUILDER) | $(DIST)
 
 clean:
 	rm -rf $(DIST) pkg/pillar/Dockerfile pkg/qrexec-lib/Dockerfile pkg/qrexec-dom0/Dockerfile \
-		images/installer.yml $(ROOTFS_YML_xen) $(ROOTFS_YML_acrn)
+		pkg/rsyslog/Dockerfile images/installer.yml $(ROOTFS_YML_xen) $(ROOTFS_YML_acrn)
```

Comment:
Hey @gkodali-zededa since we've agreed to drop dependecnies -- lets also rename Dockerfile.in back into Dockerfile and drop this line

Reply:
Done